package.json

```diff
@@ -51,7 +51,7 @@
     "@storybook/react": "8.2.9",
     "@storybook/react-vite": "8.2.9",
     "@testing-library/dom": "10.4.0",
-    "@testing-library/jest-dom": "6.4.5",
+    "@testing-library/jest-dom": "6.6.3",
```

Comment:
is it necessary to change the version of jest?

Reply:
"No, there is no need to change the jest-dom version. It was an accidental change. I will proceed to revert it to maintain consistency with the rest of the project."